spec/system/evaluations_spec.rb

```diff
@@ -1,65 +1,89 @@
+    end
+    let!(:assignment) { create(:evaluator_submission_assignment, evaluator: evaluator, submission: submission) }
 
-  before do
-    system_login_user(evaluator)
-  end
+    before do
+      system_login_user(evaluator)
+    end
 
-  it 'displays the evaluation form instructions' do
-    visit new_submission_evaluation_path(submission)
+    context "with a gov email" do
+      before do
+        evaluator.update(email: generate_user_email(type: :gov))
+      end
 
-    expect(page).to have_content(evaluation_form.instructions)
-  end
+      it 'displays the evaluation form instructions' do
+        visit new_submission_evaluation_path(submission)
 
-  it 'saves the form as a draft' do
-    visit new_submission_evaluation_path(submission)
+        expect(page).to have_content(evaluation_form.instructions)
+      end
 
-    save_evaluation_draft
+      it 'saves the form as a draft' do
+        visit new_submission_evaluation_path(submission)
 
-    expect(page).to have_content('Evaluation saved as draft')
-  end
+        save_evaluation_draft
 
-  it 'validates presence of required fields' do
-    visit new_submission_evaluation_path(submission)
+        expect(page).to have_content('Evaluation saved as draft')
+      end
 
-    complete_evaluation
+      it 'validates presence of required fields' do
+        visit new_submission_evaluation_path(submission)
 
-    expect(page).to have_content("prohibited this evaluation from being saved")
-  end
+        complete_evaluation
 
-  it 'allows entering scores for evaluation criteria' do
-    skip "Add calculated and total score display tests"
-    visit new_submission_evaluation_path(submission)
+        expect(page).to have_content("prohibited this evaluation from being saved")
+      end
 
-    fill_in_all_scores
+      it 'allows entering scores for evaluation criteria' do
+        skip "Add calculated and total score display tests"
+        visit new_submission_evaluation_path(submission)
 
-    # TODO: Fix this when updating test
-    total_score = 0
-    expect(page).to have_content(total_score)
-  end
+        fill_in_all_scores
+
+        # TODO: Fix this when updating test
+        total_score = 0
+        expect(page).to have_content(total_score)
+      end
+
+      it 'submits the form and marks the evaluation as complete' do
+        visit new_submission_evaluation_path(submission)
 
-  it 'submits the form and marks the evaluation as complete' do
-    visit new_submission_evaluation_path(submission)
+        fill_in_all_scores
+        complete_evaluation
 
-    fill_in_all_scores
-    complete_evaluation
+        expect(page).to have_content('Evaluation Complete')
+      end
 
-    expect(page).to have_content('Evaluation Complete')
+      it 'shows the submission details panel' do
+        visit new_submission_evaluation_path(submission)
+        expect(page).to have_css('[data-controller="hotdog"]')
```

Comment:
might be good to expect the page does/does not have the submitter's PII such as their email address or name on the page instead of the layout element